common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -200,6 +200,15 @@ class CapaFields(object):
         scope=Scope.settings
     )
 
+    @property
+    def non_editable_metadata_fields(self):
```

Comment:
I think it'll work, if you use `self.__class__.data` (or `self.fields['data']`), rather than `EditingDescriptor.data` (or `CapaDescriptor.data`).